src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  cursor: pointer;
+  margin-right: auto;
+  margin-left: 50px;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+}
+
+.nav__list a {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  margin-left: 16px;
+}
+
+.nav__list li a {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__list li {
+  display: inline-block;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  text-transform: uppercase;
+}
+
+.is-active {
+  margin-left: 0;
+  color: #00acdc;
+  position: relative;
+}
```

Comment:
you reset the margin here only because this is the first element, but not because it's an active one. look what happens if I make the second element active
![Screenshot 2022-03-21 at 18 57 11](https://user-images.githubusercontent.com/79282749/159325845-42880974-c861-4a39-ac1f-52a0aaefb237.jpg)
the design is broken. this margin 0 should belong to the first child, not to the is-active element